CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Minor: Fixed tag parsing for consecutive escaped characters. (#3711)
 - Minor: Prevent user from entering incorrect characters in Live Notifications channels list. (#3715)
 - Minor: Fixed automod caught message notice appearing twice for mods. (#3717)
+- Bugfix: Pasting to search in emote popup window will now ignore whitespaces. (#3730)
```

Comment:
```suggestion
- Minor: Prevent user from entering incorrect characters in Live Notifications channels list. (#3715, #3730)
- Minor: Fixed automod caught message notice appearing twice for mods. (#3717)
```
I'm not sure if this needs a separate changelog entry, while it does add an extremely useful functionality, it could be covered under the original(?)